docs/tutorials/snippets/typeorm/typeorm-async-provider.ts

```
@@ -1,19 +1,16 @@
 const CONNECTION_NAME = "default"; // change the name according to your server configuration
 
-registerProvider({
-  provide: CONNECTION,
-  deps: [Configuration],
-  async useAsyncFactory(configuration: Configuration) {
-    const settings = configuration.get<ConnectionOptions[]>("typeorm")!;
+export type CONNECTION = Connection; // Set alias types (optional)
+
+export const CONNECTION = injectable(Symbol.for("CONNECTION"))
+  .asyncFactory(async () => {
+    const settings = constant<ConnectionOptions[]>("typeorm")!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)